Wireframe/css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+    font-family: Arial, sans-serif;
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    background-color: #6e6b6b;
+    color: white;
+    text-align: center;
+    padding: 20px;
+}
+
+header h1 {
+    font-size: 2em;
+}
+
+header p {
+    font-size: 1.2em;
+}
+
+main {
+    background-color: white;
+    flex: 1;
+    padding: 20px;
+}
+
+.articles {
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+}
+
+.articles article:first-child {
```

Comment:
Do you need all this specificity? What would happen if you removed the class?

Reply:
Thanks. Initially, I used the :first-child selector to highlight the first article, but I didn't  apply it effectively. I've now tried to  fix this. I updated the styling with a different background, more padding, a  box shadow, and a rounded border to make it stand out more clearly.